### DISCUSSION TITLE
How to remove border from tabs

### DISCUSSION THREAD

**Discussion by ThomasTopi** [rdbende/Sun-Valley-ttk-theme#50] [Q&A] [answered]
Hey, it is possible to remove this border from the selected tab?
Or there is workaround for it?
Thank you!

![image](https://user-images.githubusercontent.com/113641099/192795366-f494eee3-6196-4b2a-8ea6-5c2aa4b6eef9.png)

**Comment by rdbende** [comment, marked as answer]
That dashed border is the keyboard focus marker, so I don't recommend you to remove it.
However, if you want, you can add `takefocus=False` when creating the notebook, and it will disable keyboard focus for the notebook tabs.

**Reply by ThomasTopi** [reply]
It works, many thanks!

**Reply by RedLeafSystems** [reply]
Just saying you can also do this for buttons and other components too.

**Comment by Akascape** [comment]
@rdbende the blue color border is still not removed after passing `takefocus=0`,
It is still appearing after **selecting the tab.** Is there any other method to remove that dotted outline?

My code:
```python
from tkinter import ttk
import tkinter
import sv_ttk

root = tkinter.Tk()
root.geometry("400x400")

notebook = ttk.Notebook(root, takefocus=False)
notebook.pack(expand=True, fill="both", padx=10, pady=10)

tab_1 = ttk.Frame(notebook, takefocus=False)
notebook.add(tab_1, text=f"Tab 1")

sv_ttk.set_theme("dark")

root.mainloop()
```
Platform: windows 11

**Reply by Akascape** [reply]
I tried this method here, and it is working: https://www.tutorialspoint.com/how-to-remove-ttk-notebook-tab-dashed-line-tkinter
